main.go

```diff
@@ -40,109 +48,37 @@ func main() {
 		dbPassword = "@"
 	}
 
-	var err error
+	// var err error
 	connectionString := dbUser + ":" + dbPassword + "@tcp(127.0.0.1:3306)/" + defaultDb + "?charset=utf8&parseTime=True&loc=Local"
 	// connectionString := "host="+defaultHost+" port="+defaultPort+" user="+defaultUser+" dbname="+defaultDb+" sslmode=disable"
-	db, err = gorm.Open("mysql", connectionString) // TODO:: enable ssl
-	defer db.Close()
+	dbHandle, err := gorm.Open("mysql", connectionString) // TODO:: enable ssl
+	defer dbHandle.Close()
 
 	if err != nil {
 		fmt.Println("There was an error opeing the db: ", err)
 		// TODO :: should exit right away
 	}
 
-	runMigration()
+	runMigration(&realTacitDB{gormDB: dbHandle})
```

Comment:
Might build a `realTacitDB` once here so we don't have to do it again on line 68